cli/daemon/run.go

```diff
@@ -212,6 +214,20 @@ func (s *Server) Test(req *daemonpb.TestRequest, stream daemonpb.Daemon_TestServ
 
 	testResults := make(chan error, 1)
 	go func() {
+		defer func() {
+			if recovered := recover(); recovered != nil {
+				var err error
+				switch recovered := recovered.(type) {
+				case error:
+					err = recovered
+				default:
+					err = fmt.Errorf("%v", recovered)
+				}
+				log.Err(err).Msg("panic during test run")
+				testResults <- fmt.Errorf("panic occured witihn Encore during test run: %w\n%s", recovered, errlist.GetStack())
+			}
+		}()
+
```

Comment:
Previously a panic within Encore's codebase during a call to `encore test` would crash the daemon without logging anything to the CLI. This change enables that logging and keeps the daemon running.